GeoscanDecoder/geoscan.py

```diff
@@ -149,16 +148,21 @@ def set_merge_mode(self, val):
 
     def generate_fid(self):
         if not (self.current_fid and self.merge_mode):
-            self.current_fid = f'GEOSCAN_{dt.datetime.now()}'.replace(' ', '_').replace(':', '-')
+            self.current_fid = f"GEOSCAN_{dt.datetime.now()}".replace(" ", "_").replace(
+                ":", "-"
+            )
         return self.current_fid
 
     def force_new(self):
+        f = self.files.get(self.current_fid)
+        if f:
+            f.close()
```

Comment:
Good!

Reply:
Ok, but I had to create a .vscode directory in the project and add it to .gitignore to disable autoformatting.